src/intrinsics/prepack/utils.js

```diff
@@ -102,7 +104,7 @@ export function createAbstract(
     } else {
       realm.saveNameString(name);
     }
-    result = AbstractValue.createFromTemplate(realm, buildExpressionTemplate(name), type, [], kind);
+    result = AbstractValue.createFromTemplate(realm, buildExpressionTemplate(name), type, values, kind);
```

Comment:
This is not needed for this PR, but I noticed it while debugging and it seems worth fixing.